src/core/state.go

```diff
@@ -874,6 +874,19 @@ func (state *BuildState) queueTarget(label, dependent BuildLabel, rescan, forceB
 	return nil
 }
 
+func (state *BuildState) QueueTestTarget(target *BuildTarget) {
+	state.queueTestTarget(target)
```

Comment:
I guess we should add a TODO (and raise an issue?) to properly look into making this async.